### PR TITLE
Update search exact match count

### PR DESCRIPTION
**Pull request by doug-lovett**
Signed-off-by: Doug Lovett <doug@diamante.ca>

*Issue #:* /bcgov/entity#10695

*Description of changes:*
- 10205 update cover letter report address format. 
- 10695 report and search history summary update exact match count: number of results that match, not financing statements
- Debtor business search query update from Bob.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

**Review by jordiwes — APPROVED** [review]
LGTM

**Review by kialj876 — APPROVED** [review]

File: ppr-api/src/ppr_api/models/search_utils.py [hunk 1]
```diff
@@ -147,10 +150,10 @@
        r.registration_number AS base_registration_num,
        CASE WHEN p.last_name = :query_last AND
                  p.first_name = :query_first AND
-                 p.middle_initial = :query_middle THEN 'EXACT' ELSE 'SIMILAR' END match_type,
+                 p.middle_initial = 'E' THEN 'EXACT' ELSE 'SIMILAR' END match_type,
```
Comment (kialj876) [inline ppr-api/src/ppr_api/models/search_utils.py]: I think this might be a typo? Probably want this to match what they queried right?

**Review by kialj876 — COMMENTED** [review]
Hey Doug this looks good except one thing I mentioned below

File: ppr-api/src/ppr_api/models/search_utils.py (hunk 1 above)
Comment (doug-lovett) [reply ppr-api/src/ppr_api/models/search_utils.py]: Thanks Kial, yes I do. Updated.

**Review by kialj876 — APPROVED** [review]

**Pull request merged**